persistent-database-url.cabal

```diff
@@ -18,25 +18,24 @@ build-Type:             Simple
 library
   default-language:     Haskell2010
   hs-source-dirs:       src
-  ghc-options:          -Wall
-  exposed-modules:      Web.Heroku.Persist.Postgresql
+  ghc-options:          -Wall -Werror
```

Comment:
Unfortunately, Hackage won't allow a package with this. They consider it too risky that someone bringing you in as a dependency could run into trouble if your package suddenly errors on a warning during compilation (maybe on a different GHC version than you expect).

The upload error will tell you to make it conditional on a package flag (e.g. `-fdevelopment`), but I find that annoying and instead choose to keep it off by default but compile with `-Werror` on CI to catch things at PR time.